### PR TITLE
feat: dual rewards

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**

### CODE REVIEW

File: v2/features/dill/RevenueStats.tsx [hunk 1]
```diff
@@ -76,6 +81,21 @@ const RevenueStats: FC<Props> = ({ dill }) => {
...
+                <div className="flex justify-between items-end">
+                  <div className="font-bold text-foreground text-xs mr-2">ETH:</div>
+                  <div className="text-foreground text-xs">{formatPercentage(ethRewards)}</div>
+                </div>
+              }
+              {
+                <div className="flex justify-between items-end">
+                  <div className="font-bold text-foreground text-xs mr-2">PICKLE:</div>
+                  <div className="text-foreground text-xs">{formatPercentage(pickleRewards)}</div>
+                </div>
```
Comment (janklimo) [inline v2/features/dill/RevenueStats.tsx:96]: Screen Shot 2022-05-18 at 21 00 48

PICKLE shows up as 0

File: v2/features/dill/flows/hooks.ts [hunk 2]
```diff
@@ -38,9 +38,23 @@ export const useDistributorContract = (address: string) => {
...
+  const blocksPerMinute: Record<number, number> = {
+    1: 4.6, // eth
+    137: 30, // poly
+    42161: 20, // arb
+    66: 20, // oec
+    1666600000: 30, // harmony
+    1285: 4.3, // moonriver
+    25: 5.7, // cronos
+    1313161554: 1, // aurora
+    1088: 17, // aurora
```
[the remark this answers is not in the document]
Comment (janklimo) [reply v2/features/dill/flows/hooks.ts:53]: One more thing: do we need to work with sidechains at all? Isn't APY determined by mainnet only?

**Review by janklimo — CHANGES_REQUESTED** [review]
@UrAvgDeveloper left some comments, good start. For now, my Dill balance and lock end date, rewards are not showing on this branch. I didn't look into why.
Screen Shot 2022-05-18 at 21 02 03

**Comment by UrAvgDeveloper** [comment]
> @UrAvgDeveloper left some comments, good start. For now, my Dill balance and lock end date, rewards are not showing on this branch. I didn't look into why. Screen Shot 2022-05-18 at 21 02 03
> 
> Two more observations:
> 
> 1. v1 Dill page breaks
> 2. This doesn't update the v2 chart. Will you do it as a part of this PR?

@janklimo thanks for the comments, looking into them. DILL balance shown is for Fee Distributor V2. That is the reason you do not see any DILL balance (You might have in V1).

Will look into the v1 DILL page breaking issue.

Yes I have not updated the chart to consider ETH rewards yet but they should work fine for Pickle rewards as pf-core is configured accordingly.

File: v2/features/dill/RevenueStats.tsx (hunk 1 above)
Comment (UrAvgDeveloper) [reply v2/features/dill/RevenueStats.tsx:96]: This is just waiting for the latest pf-core release. It should come up once that is done.

File: v2/features/dill/DillAmount.tsx [hunk 3]
```diff
@@ -9,53 +9,66 @@ import GetDillModal from "v2/features/dill/GetDillModal";
...
+  const pickleLocked = parseFloat(formatEther(userDill.pickleLocked || "0"));
+  const pickleBalance = parseFloat(formatEther(pickles[ChainNetwork.Ethereum] || "0"));
   return (
     <>
       <div>
         <aside className="border border-foreground-alt-500 grow font-title rounded-lg tracking-normal p-4">
           <h1 className="font-medium text-foreground-alt-200 text-base leading-5">
             {t("v2.dill.dillAmount")}
-            {dillBalance && (
+            {/* {dillBalance && (
```
Comment (larrythecucumber321) [inline v2/features/dill/DillAmount.tsx:35]: This doesn't seem necessary to remove, was there a reason for doing so?

File: v2/features/dill/RevenueStats.tsx [hunk 4]
```diff
@@ -100,6 +118,21 @@ const RevenueStats: FC<Props> = ({ dill }) => {
                   />
                 </div>
               </div>
+              <div className="flex items-center font-bold text-sm">
+                <span className="text-foreground mr-1">
+                  {formatNumber(upcomingDistribution.weeklyEthAmount)}
+                </span>
+                <div className="inline-block w-6 h-6">
+                  <Image
+                    src="/ethereum.png"
```
Comment (larrythecucumber321) [inline v2/features/dill/RevenueStats.tsx:127]: Please change to `/tokens/eth.png`. Going forward after the release of UI v2, the token names/labels will rely on the naming in pf-core's `ExternalTokenModel`, and the `public` assets will mainly fall into either `/tokens` or `/protocols`

File: v2/features/dill/UnlockDate.tsx [hunk 5]
```diff
@@ -1,32 +1,64 @@
...
 import { IUserDillStats } from "picklefinance-core/lib/client/UserModel";
 import dayjs from "dayjs";
-
+import localizedFormat from "dayjs/plugin/localizedFormat";
+import duration from "dayjs/plugin/duration";
 import IncreaseLockDateModal from "v2/features/dill/IncreaseLockDateModal";
 import Button from "v2/components/Button";
-import DillCard from "./DillCard";
-
+dayjs.extend(duration);
```
Comment (larrythecucumber321) [inline v2/features/dill/UnlockDate.tsx:9]: These plugins are already loaded into `dayjs` in `utils/day.js`, you can import it using `import dayjs from "util/dayjs";`

File: v2/features/farms/flows/harvest/HarvestFlow.tsx [hunk 6]
```diff
@@ -197,11 +247,20 @@ const HarvestFlow: FC<Props> = ({
               <p>
                 {t("v2.farms.harvesting")}
                 <span className="text-primary ml-2">
-                  {roundToSignificantDigits(pendingRewardAmount, 3)} {rewardName}
-                </span>
+                  {
+                    `${roundToSignificantDigits(pendingRewardAmount, 3)} ${rewardName}${isEthClaimable
```
Comment (larrythecucumber321) [inline v2/features/farms/flows/harvest/HarvestFlow.tsx:251]: Please use a localized "and"

**Review by larrythecucumber321 — APPROVED** [review]

**Pull request merged**